### PR TITLE
Simplify csproj files

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: Source/Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (stsrki) [inline Source/Directory.Build.props:1]: Can this be part of `Build\Blazorise.props`? Or is `Directory.Build.props` some special rule for VS to pick up when building projects?

File: Source/Extensions/Blazorise.Animate/Blazorise.Animate.csproj [hunk 2]
```diff
@@ -3,20 +3,10 @@
   <Import Project="..\..\..\Build\Blazorise.props" />
 
   <PropertyGroup>
-    <PackageId>Blazorise.Animate</PackageId>
```
Comment (stsrki) [inline Source/Extensions/Blazorise.Animate/Blazorise.Animate.csproj:6]: What happens with PackageId? Is it going to be automatically generated based on project name?

File: Source/Directory.Build.props [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<Project>
+  <PropertyGroup>
+    <PackageLicenseFile>LICENSE.md</PackageLicenseFile>
+    <PackageIcon>Blazorise.png</PackageIcon>
+    <PackageProjectUrl>https://blazorise.com/</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/Megabit/Blazorise</RepositoryUrl>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
```
Comment (stsrki) [inline Source/Directory.Build.props:9]: NoWarn is now enabled for all projects. I still want them for Blazorise code projects and other projects that are directly used.

File: Source/Directory.Build.props (hunk 1 above)
Comment (SimonCropp) [reply Source/Directory.Build.props:1]: VS walks up the directory tree and looks for Directory.Build.props

File: Source/Extensions/Blazorise.Animate/Blazorise.Animate.csproj (hunk 2 above)
Comment (SimonCropp) [reply Source/Extensions/Blazorise.Animate/Blazorise.Animate.csproj:6]: yep it defaults to the project name

**Comment by stsrki** [comment]
I used existing `Blazorise.props` for shared properties instead of `Directory.Build.props`. This feature was not working in the previous version of VS. It's good they finally fixed it.

I also tested on `dotnet pack` and it seems there are no issues with new settings.

**Pull request merged**